js/index.js

```diff
@@ -1 +1,112 @@
 // Your code goes here
+
+TweenMax.to('.car', 5, {x: 600});
```

Comment:
It's good to see you experimenting with GSAP, it can be very powerful, but it can also be overkill for simple animations.